src/genlab_bestilling/admin.py

```diff
@@ -5,20 +5,36 @@
 )
 
 from .models import (
+    AnalysisOrder,
+    AnalysisResult,
     AnalysisType,
     Area,
+    EquimentOrderQuantity,
     EquipmentBuffer,
+    EquipmentOrder,
     EquipmentType,
+    ExtractionOrder,
+    ExtractionPlate,
+    ExtractPlatePosition,
     Genrequest,
     Location,
     LocationType,
     Marker,
+    Order,
```

Comment:
This is a PolymorphicModel. I do not think we want it in the admin.

Reply:
Understood, it's omitted